setup.py

```diff
@@ -20,12 +20,13 @@ def get_version():
     version=get_version(),
     packages=["psqlgraph"],
     install_requires=[
-        'psycopg2~=2.7',
+        'psycopg2-binary~=2.8',
         'sqlalchemy~=1.3',
         'py2neo~=2.0',
         'progressbar',
         'avro~=1.7',
         'xlocal~=0.5',
-        'requests~=2.5'
+        'requests~=2.5',
+        'six>=1.12.0',
```

Comment:
since we are not trying to keep python 2 support, i don't think this dependency is necessary - we only use string_types (str), integer_types (int) and ensure_str (can be replaced by str.encode)

Reply:
I think since we'll eventually cut over the gdc's psqlgrah and they are keeping python2 support, we should just leave `six` in so our code is consistent. Thoughts?

Reply:
yeah sure